service/statistics/AnalyticsEvents.js

```diff
@@ -565,3 +565,17 @@ export const createBridgeChannelClosedEvent = function(code, reason) {
 export const createTtfmEvent = function(attributes) {
     return createConnectionStageReachedEvent('ttfm', attributes);
 };
+
+/**
+ * Creates an event with performance metrics information. Each element in the metrics
+ * array will have a name and duration.
+ *
+ * @param {Array<object>} metrics - Array of metrics objects.
+ */
+export const createPerfMetricsEvent = function(metrics) {
+    return {
```

Comment:
I'm not sure if sending separate events per metric entity would be better. I'm not really sure if we would be able to filter those metrics in Amplitude since all the info (name, timestamp, etc) is in the array. 

Maybe we can send separate events with the action field the name of the mark and then all the values? WDYT?

Reply:
Right now metrics will come in batches, not sure if 1 event per metric would be overkill. But doable. If we opt for individual events I can flatten the thing and make it like so:

```
    return {
        type: TYPE_OPERATIONAL,
        action: 'perf-metrics',
        metric: 'jitsi.foo.bar',
        duration: xxxxx
    };
```

Do you think we should go for that?